homeworks/nemethmate/8_streamlit/forecast.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+def get_forecast(city, api_key):
```

Comment:
Na pontosan erre gondoltam feljebb, amikor mondtam hogy kell egy function amit cache-elni tudsz.
Alakítsd át ezt a functiont úgy, hogy az endpoint (weather vagy forecast) az url-ben egy paraméter legyen a functionnek. Így a city mellett kelleni fog ennek a functionnek egy endpoint paraméter is. Ezt pedig tudod cache-elni úgy ahogy az órán tanultuk.